src/host/pico_multicore/include/pico/multicore.h

```diff
@@ -20,11 +20,13 @@ void multicore_launch_core1_raw(void (*entry)(void), uint32_t *sp, uint32_t vect
 
 bool multicore_fifo_rvalid(void);
 bool multicore_fifo_wready(void);
-void multicore_fifo_push(uint32_t data);
+void multicore_fifo_push_blocking(uint32_t data);
+bool multicore_fifo_push_timeout_us(uint32_t data, uint64_t timeout_us);
```

Comment:
Maybe these also need adding to https://github.com/raspberrypi/pico-host-sdl/blob/master/sdl_video.c#L1618 ? :shrug: